subcommands/keys/resign_root.go

```diff
@@ -82,5 +82,5 @@ func doResignRoot(cmd *cobra.Command, args []string) {
 		os.Exit(1)
```

Comment:
this is on the commit message. Can you explain what's unsafe about copy-targets?

Reply:
changed in force-push to be less opinionated